src/hooks/force-update.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Linking, Platform } from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+
+export const useValidateAppVersion = () => {
+  const [showModal, setShowModal] = useState(false);
+
+  useEffect(() => {
+    const checkAndUpdate = async () => {
+      const updated = await isUpdated();
+      if (!updated) {
+        setShowModal(true);
+      }
+    };
+    checkAndUpdate();
+  });
+
+  return { showModal, openStore };
+};
+
+const isUpdated = async (): Promise<boolean> => {
+  remoteConfig().setDefaults({
+    minimum_version: '',
+  });
+
+  const fetchedRemotely = await remoteConfig().fetchAndActivate();
+
+  if (fetchedRemotely) {
+    const remoteVersion = remoteConfig().getValue('').asString();
```

Comment:
It wouldn't be more convenient to put this value and the default minimum version on the env.ts file instead of creating another enum?

Reply:
These are different things, we want the enum to be a set of constant values and use it to `getValue`s from the remote config to avoid misspellings or typos.

As for the default `minimun_version` could be in the env file but I don't see the benefit if the default value could be null or a falsy to avoid the function comparison running.

Reply:
Done